.gitignore

```diff
@@ -16,4 +16,6 @@
 
 /config/master.key
 
-coverage/
\ No newline at end of file
+coverage/
+# Ignore master key for decrypting credentials and more.
+/config/master.key
```

Comment:
See line 17

Reply:
ah i see it - better than not having at all I guess!